tests/crypto_aes/main.c

```diff
@@ -126,37 +126,71 @@ parse_testcase(const struct testcase testcase,
 	return (1);
 }
 
+struct perftest_cookie_aes {
+	struct crypto_aes_key * key_exp;
+	uint8_t plaintext_arr[16];
+};
+
+static int
+perftest_init(void * cookie, uint8_t * buf, size_t buflen)
+{
+	struct perftest_cookie_aes * pca = cookie;
+
+	/* Set the plaintext input. */
+	memcpy(buf, pca->plaintext_arr, buflen);
```

Comment:
I feel like this needs a "Note that `buflen` will always be equal to 16" or something like that given that `plaintext_arr` is fixed at 16 bytes.

Maybe an `assert` actually?